src/components/ChatLog.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatEntry.css';
+// import PropTypes from 'prop-types';
+
+const ChatLog = ({ initialMessage }) => {
+  const Vladimir = [];
+  const Estragon = [];
+  for (const message of initialMessage) {
+    if (message.sender === 'Vladimir') {
+      Vladimir.push(
+        <ChatEntry
+          key={message.id}
+          id={message.id}
+          body={message.body}
+          sender={message.sender}
+          liked={message.liked}
+          timeStamp={message.timeStamp}
+          // updateLikes={updateLikes}
+        />
+      );
+    } else {
+      Estragon.push(
+        <ChatEntry
+          key={message.id}
+          id={message.id}
+          body={message.body}
+          sender={message.sender}
+          liked={message.liked}
+          timeStamp={message.timeStamp}
+          // updateLikes={updateLikes}
+        />
+      );
+    }
+  }
```

Comment:
This code will only work if the users are named Vladimir and Estragon. We should write our code so that it works in the general case - not just on one input. Note that in the tests there are different users.